Automattic/wp-calypso
# Upgrades: Introduce visuals for wordads instant activation

This provides visuals for WordAds-Instant Activation.
It will show "WordAds" element in sidebar for business users and let them activate WordAds instantly

Relevant sections are blocked, only outline is showed.
Copy probably could use some tweaks

# Testing

- Check out this branch
- Connect your sandbox
- Apply Phabricator DIFF D1761
- Create a new premium/business site
- You should have WordAds in Sidebar now
- After togling "Activate" in WordAds settings you should receive an email and now you thould have WordAds enabled. 

##### TODO

- [x] Connect toggle to endpoint
- [x] Reload site in sitesList once WordAds is activated for a site

## Earnings

![zrzut ekranu 2016-05-11 o 09 55 20](https://cloud.githubusercontent.com/assets/3775068/15174096/2307fe54-1760-11e6-9d7f-7bf9271a4cff.png)

## Settings

![zrzut ekranu 2016-05-11 o 09 55 27](https://cloud.githubusercontent.com/assets/3775068/15174102/2b902af6-1760-11e6-8ccf-4dd9086deff6.png)


CC @rralian @mtias @adambbecker @egill @scruffian @dbspringer @drw158
